src/webui/www/private/scripts/client.js

```diff
@@ -68,6 +69,10 @@ function genHash(string) {
     return hash;
 }
 
+function getSyncMainDataInterval() {
+    return (!customSyncMainDataInterval) ? serverSyncMainDataInterval : customSyncMainDataInterval;
```

Comment:
the parentheses aren't required and also you can switch the true-false cases to save an NOT operation:
`return customSyncMainDataInterval ? customSyncMainDataInterval : serverSyncMainDataInterval;`